### PR TITLE
Change ReleaseScript and Links, Change test name

### PR DESCRIPTION
**Pull request by RomanenkoVladimir**
##Tech/1117/rename master branch to main branch

_**Please read the [CONTRIBUTING.md](https://github.com/MaibornWolff/codecharta/blob/master/CONTRIBUTING.md) before opening a PR.**_

Issue: #1117 

## Description
Change the release scripts and the github-pages links so that everything should continue working after the branch name change.
Change a few namings that are connected to the request as well.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Changed
 
+- Rename master branch to main for a more inclusive naming
```
Comment (NearW) [inline CHANGELOG.md:18]:
```suggestion
- Rename master branch to main for a more inclusive naming #1117 
```

Always add the issue number, if there is one. :)

**Comment by BridgeAR** [comment]
All open PRs also have to be set to main as destination branch.

**Comment by NearW** [comment]
Merged #1119
This branch has some conflicts now since make_release.sh has been replaced with a python script.
Please update the python accordingly

**Comment by RomanenkoVladimir** [comment]
> 
> 
>     * I think something like this deserves a changelog entry to bring more attention to the topic in general.
> 
> 
> I pushed a commit, because somehow some line breaks got missing in a README

Oh

File: script/make_release.sh [hunk 2]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
```
Comment (NearW) [inline script/make_release.sh:1]: This file can be deleted

**Comment by BridgeAR** [comment]
There's a conflict that has to be resolved.

**Pull request merged**